src/AspNet.Security.OAuth.WorkWeixin/AspNet.Security.OAuth.WorkWeixin/AspNet.Security.OAuth.WorkWeixin.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net5.0</TargetFrameworks>
+  </PropertyGroup>
+
+
```

Comment:
Missing `Description`, `Authors` and `PackageTags`. Please add using any other provider `.csproj` file as an example for what to do.